src/lib/snack-bar/simple-snack-bar.scss

```diff
@@ -11,14 +11,19 @@ $mat-snack-bar-button-margin: 48px !default;
 }
 
 .mat-simple-snackbar-action {
-  @include mat-button-reset;
-
-  background: none;
+  display: flex;
+  flex-direction: column;
   flex-shrink: 0;
-  margin-left: $mat-snack-bar-button-margin;
+  justify-content: space-around;
+  margin: -8px 0 -8px $mat-snack-bar-button-margin;
```

Comment:
Use the variable here?

Reply:
I can use a variable, however they are different margins so to speak.

The top and bottom margins are `-8px` so that the total line height adds back up to `36px` (20 + 8 + 8).  The left and right margins are for spacing within the container according to spec.


Should I create another variable?  Or comment to explain it?

Reply:
Having another variable should be fine.

Reply:
Done